docker-compose.yml

```
@@ -31,8 +35,30 @@ services:
     command: |
       sh -c "npm install && npm run dev"
     networks:
-      - frontend
+      - client
     ports:
       - '3000:3000'
     volumes:
       - ./client/:/usr/src/client
+
+  postgres:
+    image: postgres
+    restart: always
+    environment:
+      DB_NAME: neoacademy
+      POSTGRES_USER: neoacademy
+      POSTGRES_PASSWORD: neoacademy
```

Wouldn't be better to use environment variables here, instead of the plain data?

We could use something like this:
```suggestion
      POSTGRES_USER: ${PG_NA_USER}
      POSTGRES_PASSWORD: ${PG_NA_PASS}
```

The same at the [line 60](https://github.com/NeoAssist/NeoAcademy/pull/20/files#diff-4e5e90c6228fd48698d074241c2ba760R60).

---

@caioamaralgit, We can leave it as environment variables in docker-compose.yml, however, I don't know if it is everyone who will create these respective variables in their environments, if any dev does not have the variables `${PG_NA_USER}` and `${PG_NA_PASS}`, the bank can start (or not start at all) without credentials.

A more interesting approach would be to leave the environment variables as "initializers" and fallback, using hardcoded credentials

---

So, we could use a default for these environment variables, using [paramenter expansion](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html) like this: 

```suggestion
      POSTGRES_USER: ${PG_NA_USER:-neoacademy}
      POSTGRES_PASSWORD: ${PG_NA_PASS:-neoacademy}
```

This way the user can configure theses credentials as desire, using `neoacademy` as default.